docs/standard/index.md

```diff
@@ -1,60 +1,48 @@
 ---
-title: .NET Guide
-description: Learn about .NET.
+title: .NET Standard guide
```

Comment:
the folder name is misleading. It's just .NET Guide. We tried to come up with better names but never reached a consensus. We could try again. Some options were .NET Fundamentals, .NET Primer, etc.

```suggestion
title: .NET guide
```